rmgpy/data/kinetics/common.py

```diff
@@ -33,27 +33,11 @@
 in this subpackage.
 """
 
-from rmgpy.data.base import DatabaseError, LogicNode
-from rmgpy.reaction import Reaction, ReactionError
```

Comment:
I don't believe so, which is why i removed it. 

My IDE said 'ReactionError imported bu unused', so down the drain it went.